frontend/shared-components/state-input/state-input.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FormEvent, useEffect, useState } from "react"
```

Comment:
I'm curious why we have two state-input.tsx files, one in compositions and one in shared-components

Reply:
Oof that was an oversight resolving merge conflicts - great catch! 
I think we want the one that's in shared-components as it will likely be used across different pages so I'll remove the compositions one